### PR TITLE
[DOCS] Add documentation for DB tables

### PR DESCRIPTION
**Pull request by MikeVedsted**
Adding documentation and overview for db tables based on [this diagram](https://dbdiagram.io/d/5fc4ecd83a78976d7b7deb48)

This is for issue #34

### CODE REVIEW

**Review by nannadao — APPROVED** [review]
Looking great! 👍🏼

**Review by Rostislav013 — APPROVED** [review]

**Review by MarinaFroes — APPROVED** [review]

File: docs/db/tables.sql [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+CREATE TABLE admin(
+  admin_id SERIAL PRIMARY KEY,
+  userk uuid REFERENCES users(user_id) ON DELETE CASCADE
+);
+
+CREATE TABLE banned_user(
+  banned_user_id SERIAL PRIMARY KEY,
+  userk uuid REFERENCES users(user_id) ON DELETE CASCADE
+);
```
Comment (nannadao) [inline docs/db/tables.sql:116]: Here the table for user is `users`

File: docs/db/tables.sql [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+CREATE TABLE comment(
+  comment_id uuid DEFAULT uuid_generate_v4 () PRIMARY KEY,
+  comment text NOT NULL,
+  userk uuid REFERENCES userk(user_id) ON DELETE CASCADE,
+  event uuid REFERENCES event(event_id) ON DELETE CASCADE
```
Comment (nannadao) [inline docs/db/tables.sql:122]: Here table for user  is `userk`

**Review by nannadao — CHANGES_REQUESTED** [review]

**Review by Chiranjibichapagain — APPROVED** [review]
looks good.

**Review by nannadao — APPROVED** [review]

**Pull request merged**